programs/svm-spoke/src/event.rs

```diff
@@ -121,7 +121,6 @@ pub struct ClaimedRelayerRefund {
     pub refund_address: Pubkey,
 }
 
-// TODO: update the name of this to EmergencyDeletedRootBundle and in EVM.
```

Comment:
we have not fixed this in EVM

Reply:
done here: https://github.com/across-protocol/contracts/pull/760